docs/resources.rst

```diff
@@ -0,0 +1,39 @@
+.. _resources:
+
+Other resources and learning material
+#####################################
+
+Vyper has an active community. You can find third party tutorials,
+examples, courses and other learning material.
+
+General
+-------
+
+-  `More Vyper by Example <https://vyper-by-example.org/> by Smart Contract Engineer`__
+-  `Vyper cheat Sheet <https://reference.auditless.com/cheatsheet>`__
+-  `Vyper Hub for development <https://github.com/zcor/vyper-dev>`__
+-  `Vyper greatest hits smart contract examples <https://github.com/pynchmeister/vyper-greatest-hits/tree/main/contracts>`__
+-  `Awesome Vyper curated resources <https://github.com/spadebuilders/awesome-vyper>`__
+
+Frameworks and tooling
+----------------------
+
+- `ApeWorX - The Ethereum development framework for Python Developers, Data Scientists, and Security Professionals <https://www.apeworx.io/>`__
+- `Forge development framework <https://www.alchemy.com/dapps/foundry-forge>`__
```

Comment:
Forge doesn't have support for Vyper out of the box, probably better linking that hack that let's it work instead

Reply:
Alchemy advertised Vyper support for me, so maybe I got fooled here 🤨